flutter/android/app/build.gradle

```diff
@@ -22,10 +22,6 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-// START: FlutterFire Configuration
```

Comment:
@anhappdev I was getting some gradle errors when building with firebase.
Can you please help with the correct command to be used. mostly the build id that i used is wrong.

Reply:
Did you set up Firebase env vars as described here?
https://github.com/mlcommons/mobile_app_open/blob/master/docs/environment-setup/env-setup-firebase.md